Automattic/jetpack
# Stats: enable Odyssey Stats for WoA sites

Related: https://github.com/Automattic/jetpack/issues/35615

## Proposed changes:

- Enable Odyssey Stats for WoA (Atomic) sites

Atomic sites were using Calypso Stats before, but with the process of the untangling Calypso project going on, Atomic sites now have an option to stay within WP-Admin for (almost) everything. Atomic sites were intentionally left out when rolling out the new Stats experience (Odyssey Stats), and now it's the time to enable it.

The PR should be merged after all the parity work is finished at https://github.com/Automattic/jetpack-roadmap/issues/1204

### Other information:


## Jetpack product discussion
pfsHM7-5B-p2#comment-153

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Create an Atomic dev blog following instructions at p9o2xV-1r2-p2
* Install Jetpack and Jetpack Beta plugin
* Choose branch `update/enable-odyssey-for-woa-sites` in Jetpack Beta
* Open `/wp-admin/admin.php?page=stats`
* Ensure all the pages work okay
